sentry-raven.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "faraday", ">= 0.7.6"
   gem.add_dependency "certifi"
+  gem.add_dependency "binding_of_caller" if RUBY_VERSION >= '1.9.2' && !defined?(JRUBY_VERSION)
```

Comment:
We should re-use [the `integrations` infrastructure](https://github.com/getsentry/raven-ruby/blob/master/lib/raven/base.rb#L203) already in the project, which deals better with Gemfile ordering issues.